.circleci/config.yml

```diff
@@ -135,6 +135,23 @@ jobs:
       - *generate_base_path
       - run:
           command: npm run build
+      - *persist_to_workspace
+
+  upload_to_cloudflare:
+    resource_class: medium+
```

Comment:
don't need this since it's not that heavy

```suggestion
```

Reply:
done